connect.js

```diff
@@ -23,7 +23,8 @@ const excludedProps = ['anyTouched', 'mutator', 'connectedSource'];
 // or common props introduced by stripes-connect or redux-form
 function arePropsEqual(props, prevProps) {
   return _.isEqualWith(props, prevProps, _.after(2, (p1, p2, key) => {
-    return (_.isFunction(p1) || _.isFunction(p2) ||
+    // eslint-disable-next-line
+    return (_.isFunction(p1) || _.isFunction(p2) || _.isObject(p1) && p1['$$typeof'] || _.isObject(p2) && p2['$$typeof'] ||
```

Comment:
Would it be possible to use [`isValidElement`](https://reactjs.org/docs/react-api.html#isvalidelement) here instead?

If not, a comment and some parentheses might make the purpose of this a bit clearer.

Reply:
BTW, those linter errors are valid IMO and I don't think they should be disabled. The mixing of `&&` and `||` are specifically what I was talking about with the addition of parentheses.

Reply:
Do we need to check for `isObject` here? Could we just add `$$typeof` key to the list of excluded props?

Reply:
Just tried, it would break. As p1 and p2 are a values which can be of any type.

Reply:
@doytch, I searched for something like this initially but seems like not enough😅. `isValidElement` seems like works and it is even better to use it in case the prop, which is rather an implementation detail, would be removed or renamed.

Reply:
@VictorSoroka1 yes but we don't need to touch `p1` and `p2` in this case. Just test if the `key  ==="$$typeof"` right?

Reply:
My idea was to take an object and check if it is React element and just stop there. The object which represents React component has this prop with some value in it so I think this check will just exclude this key but not other ones.